app/controllers/passwordless/sessions_controller.rb

```diff
@@ -96,5 +96,9 @@ def find_session
         token: params[:token]
       )
     end
+
+    def session_expired(session)
```

Comment:
I think we should move the responsibility of knowing whether the session is expired to the session itself? Let's move this to a method `Session#expired?`